deploy/job-manager/deploy.sh

```diff
@@ -296,22 +327,47 @@ if [ -z $2 ]; then
 fi
 
 if [ -z $3 ]; then
-    echo -e "\nYou must specify a desired username for Job Manager UI!"
+    echo -e "\nYou must specify the url for the Cromwell instance to use with the Job Manager UI!"
     error=1
 fi
 
 if [ -z $4 ]; then
-    echo -e "\nYou must specify a desired password for Job Manager UI!"
+    echo -e "\nYou must specify whether to use Cromwell-as-a-Service with Job Manager UI!"
     error=1
 fi
 
 if [ -z $5 ]; then
-    echo -e "\nMissing the Vault token file under $HOME/.vault-token, you need to make sure you have passed in the path to the token file as the 5th argument of this script!"
+    echo -e "\nYou must specify whether to use a UI proxy!"
+    error=1
 fi
 
+if [ $5 == "true" ]; then
+    if [ -z $6 ]; then
```

Comment:
I was thinking about changing these to make them required even if USE_PROXY is false, just so that it doesn't mess up the ordering of the parameters.